coalib/core/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+Provides an API for coala's NextGen-Core
```

Comment:
Run pydocstyle on this.  I believe it will complain about the `Provides` and lack of trailing `.` .